package.json

```diff
@@ -54,6 +55,7 @@
     "reanimated-bottom-sheet": "^1.0.0-alpha.20",
     "rn-faded-scrollview": "^1.0.8",
     "tweetnacl": "^1.0.3",
+    "url": "^0.11.0",
```

Comment:
What are we using this for?

Reply:
This is needed by the `jsonschema`. I was surprised when it showed up as missing, since `jsonschema` didn't list any dependencies. Apparently `React` includes the `url` package by default, but `react-native` does not.